cdcsdk-server/cdcsdk-server-core/src/main/java/com/yugabyte/cdcsdk/server/ServerApp.java

```diff
@@ -152,6 +155,13 @@ else if (beans.size() > 1) {
         configToProperties(config, props, PROP_KEY_FORMAT_PREFIX, "key.converter.");
         configToProperties(config, props, PROP_VALUE_FORMAT_PREFIX, "value.converter.");
```

Comment:
I am guessing this is the source of the bug. A property like `cdcsdk.server.format.value.converter.schemas.enable` is getting converted to `value.converter.converter.schemas.enable`. Notice the double `converter`. 

Can you replace with the below diff with this instead?

Add the below line as well:

    configToProperties(config, props, PROP_VALUE_FORMAT_PREFIX, "value.");